src/utils/index.ts

```diff
@@ -41,6 +44,19 @@ export function omit<S>(obj: S, ...props: string[]) {
 export function byggPamUrl(aktorId: string, path: string) {
 	return `https://pam-cv-veileder${finnNaisDomene()}${path}/${aktorId}`;
 }
+export function useUrlNyPersonforvalter(fnr: string) {
```

Comment:
Tenker kanskje at denne logikken kan forenkles litt ved å lage en egen funksjon for å opprette lenken, f.eks
```
export function lagPersonforvalterLenke(aktoerIdEllerFnr: string) {
	return `https://person-forvalter${finnNaisDomene()}endreperson?aktoerId=${aktoerIdEllerFnr}`;
}
```
og flytte logikken for å hente aktorId inn i lenke-brukerprofil.tsx

Reply:
Også URLen er ikke riktig.  Den skal være sånn:- 
`https://person-forvalter.${finnNaisDomene()}/endreperson?aktoerId=${aktoerIdEllerFnr}`;

Reply:
Nå finnes lagPersonforvalterLenke, og aktorId-henting er flyttet. Lenkene er oppdatert til å bruke intern.nav.no